storage/storagedriver/select-storage-driver.md

```diff
@@ -99,13 +98,13 @@ configurations work on recent versions of the Linux distribution:
 | Docker CE on CentOS | `overlay2`                                                             | `overlay`¹, `devicemapper`², `zfs`, `vfs`         |
 | Docker CE on Fedora | `overlay2`                                                             | `overlay`¹, `devicemapper`², `zfs`, `vfs`         |
 
-¹) The `overlay` storage driver is marked deprecated in docker 18.09, and will be
-removed in a future release. Users of the `overlay` storage driver are recommended
-to migrate to `overlay2`.
+¹) The `overlay` storage driver is deprecated in Docker 18.09, and will be
```

Comment:
I'm a little confused by the usage of "Docker 18.09" here. Is it implicit that this is Engine or should we explicitly mention that?

Reply:
adding "Engine" won't hurt

Reply:
s/Docker/Docker Engine - Enterprise Thanks!